README.md

```diff
@@ -48,157 +53,127 @@ limitations under the License.
 ```
 
-**ResNet-50 Sparsified**
+[Getting Started with CLI Benchmarking](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/benchmark_model) includes examples of select inference scenarios: 
+- Synchronous (Single-stream) Scenario
+- Asynchronous (Multi-stream) Scenario
+__ __
 
-When exploring available optimized models, you can use the `Zoo.search_optimized_models` utility to find models that share a base.
-
-Try this on the dense ResNet-50 to see what is available:
+## 👩‍💻 NLP Inference | Question Answering
 
 ```python
-from sparsezoo import Zoo
-from sparsezoo.models import classification
+from deepsparse.transformers import pipeline
 
-model = classification.resnet_50()
-print(Zoo.search_sparse_models(model))
-```
+# SparseZoo model stub or path to ONNX file
+onnx_filepath="zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-aggressive_98"
 
-Output:
+qa_pipeline = pipeline(
+    task="question-answering",
+    model_path=onnx_filepath,
+    num_cores=None, # uses all available CPU cores by default
+)
 
-```shell
-[
-    Model(stub=cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-conservative), 
-    Model(stub=cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate), 
-    Model(stub=cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned_quant-moderate), 
-    Model(stub=cv/classification/resnet_v1-50/pytorch/sparseml/imagenet-augmented/pruned_quant-aggressive)
-]
+my_name = qa_pipeline(question="What's my name?", context="My name is Snorlax")
 ```
 
-We can see there are two pruned versions targeting FP32 and two pruned, quantized versions targeting INT8.
-The `conservative`, `moderate`, and `aggressive` tags recover to 100%, >=99%, and <99% of baseline accuracy respectively.
+NLP Tutorials:
+- [Getting Started with Hugging Face Transformers 🤗](https://github.com/neuralmagic/deepsparse/tree/main/examples/huggingface-transformers)
 
-For a version of ResNet-50 that recovers close to the baseline and is very performant, choose the pruned_quant-moderate model.
-This model will run [nearly 7x faster](https://neuralmagic.com/blog/benchmark-resnet50-with-deepsparse) than the baseline model on a compatible CPU (with the VNNI instruction set enabled).
-For hardware compatibility, see the Hardware Support section.
+Tasks Supported: 
+- Text Classification (Sentiment Analysis)
+- Question Answering
+- Masked Language Modeling (MLM)
 
-```python
-from deepsparse import compile_model
-import numpy
+__ __
 
-batch_size = 64
-sample_inputs = [numpy.random.randn(batch_size, 3, 224, 224).astype(numpy.float32)]
+## SparseZoo ONNX vs. Custom ONNX Models
 
-# run baseline benchmarking
-engine_base = compile_model(
-    model="zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/base-none", 
-    batch_size=batch_size,
-)
-benchmarks_base = engine_base.benchmark(sample_inputs)
-print(benchmarks_base)
+DeepSparse can accept ONNX models from two sources: 
```

Comment:
```suggestion
The DeepSparse Engine can accept ONNX models from two sources: 
```

Reply:
Noting the name of the product is "DeepSparse Engine" and the name of the repo is "deepsparse" and so we want to consistently call the product by the actual name.